airflow/www/static/css/main.css

```diff
@@ -105,7 +119,17 @@ div.container {
 @media (prefers-reduced-motion: no-preference) {
   .navbar-brand:hover .brand-logo-pinwheel {
     transform-origin: 17.66px 17.66px;
-    animation: pinSpin 1.5s linear;
+    animation: pinSpin 1.5s linear infinite;
+  }
+
+  .navbar-brand:hover .brand-logo-divider {
+    width: 5px;
+    transition: width 0.5s;
+  }
+
+  .navbar-brand:hover .brand-logo-antidivider {
+    width: 0;
+    transition: width 0.5s;
```

Comment:
Is the width still animating? I'm happy with the tiny bit more spacing all the time.